tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,189 @@
+"""CLI tests for logging.
+
+:Requirement: FIXME
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+import logging
+from robottelo import ssh
+from robottelo.test import CLITestCase
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from fauxfactory import gen_string
+log = logging.getLogger(__name__)
+
+
+class SimpleLoggingTestCase(CLITestCase):
+    """Test class for default logging to files."""
+
+    org = None
+    product = None
+
+    def setUp(self):
+        """Tests for logging to files"""
+        super(SimpleLoggingTestCase, self).setUp()
+
+        # need own org for the manifest refresh test
+
+        if SimpleLoggingTestCase.org is None:
+            SimpleLoggingTestCase.org = make_org(cached=True)
```

Comment:
Perhaps you want [`setUpClass`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass), which is called once per class (and not before each test in class)? This way you will be able to drop these conditionals.

Also, why `SimpleLoggingTestCase.org` and not `self.org`? You will have single instance of this class anyway, so it's not like you need to share this data between instances.

Reply:
Yes, thank you.

Reply:
Hello

How do I make use of the cls org in the candlepin test? Something like:

    -            result = connection.run('hammer subscription refresh-manifest --organization-id 1')
    +            result = connection.run('hammer subscription refresh-manifest --organization-id {0}'
    +                                    .format(cls.org.id))

or  cls.org['id']

Reply:
It's in the same class, right? Reference to organization should be available under `self.org`. `make_org` returns dict, so you want `self.org['id']`.

If it did not work, let me know what error did you encounter.

Reply:
This works, thanks:

     +result = connection.run('hammer subscription refresh-manifest --organization-id {0}'
     +                                    .format(self.org['id']))


However this raises another problem. The clone manifest function changes the consumer UUID so when you try to refresh it you get an error. Options that I can see: Just use default org (that might make the test disruptive if not destructive); download and use another manifest just for this test.

Thank you

Reply:
I will try to get logs from the upload event, that would solve the problem.